app/api-v1/controllers/Order/helpers.js

```diff
@@ -1,5 +1,24 @@
 const db = require('../../../db')
-const { NoTokenError, NothingToProcess } = require('../../../utils/errors')
+const { NotFoundError, NoTokenError, NothingToProcess, BadRequestError } = require('../../../utils/errors')
+
+exports.validate = async (body) => {
+  // Will add a get function at a later date to check for duplication
+  // This section checks if the order supplier does not match the supplier
+  const uniqueRecipeIDs = [...new Set(body.items)]
+
+  const recipes = await db.getRecipeByIDs(uniqueRecipeIDs)
+  if (recipes.length != uniqueRecipeIDs.length) {
+    throw new NotFoundError('recipe')
```

Comment:
This is incorrect. `NotFoundError` will return a `404` and not a `400`. Whilst the error represents something missing it's not the path itself that's missing, which is what a `404` refers to. With the errors you've removed (and I'm not entirely sold on that) this would now be a `BadRequestError` so it returns a `400`